### PR TITLE
Remove Access Token

### PR DESCRIPTION
**Pull request by KunJeongPark**
### Summary of changes

- Remove accessToken from Config
- Added showing errorView in case of missing ClientId for Native checkout flow in Demo app
- Branched off of https://github.com/paypal/iOS-SDK/pull/156 PR to resolve conflicts with refactor https://github.com/paypal/iOS-SDK/pull/157

### Checklist

~- [ ] Added a changelog entry~

### Authors
- @KunJeongPark @sshropshire

### CODE REVIEW

File: Demo/Demo/Models/ClientIDRequest.swift [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    public var method: HTTPMethod = .get
+    public var body: Data?
+
+    public var headers: [HTTPHeader: String] {
+        [
+            .accept: "application/json",
+            .acceptLanguage: "en_US"
+        ]
+    }
+}
```
Comment (jaxdesmarais) [inline Demo/Demo/Models/ClientIDRequest.swift]: If this is all just for the demo app we probably can have it be internal vs public

File: Demo/Demo/Models/ClientIDResponse.swift [hunk 2]
```diff
@@ -0,0 +1,8 @@
+public struct ClientIDResponse: Codable {
```
Comment (jaxdesmarais) [inline Demo/Demo/Models/ClientIDResponse.swift]:
```suggestion
struct ClientIDResponse: Codable {
```

File: Demo/Demo/Networking/DemoMerchantAPI.swift [hunk 3]
```diff
@@ -102,36 +102,33 @@ final class DemoMerchantAPI {
...
+        if let injectedClientID = InjectedValues.clientID {
+            return injectedClientID
         }
-        guard let token = self.accessToken else {
-            self.accessToken = await fetchAccessToken(environment: environment)
-            return self.accessToken
+        
+        guard let clientID = self.clientID else {
+            self.clientID = await fetchClientID(environment: environment)
+            return self.clientID
```
Comment (jaxdesmarais) [inline Demo/Demo/Networking/DemoMerchantAPI.swift]:
```suggestion
        guard let clientID else {
            clientID = await fetchClientID(environment: environment)
            return clientID
```

File: Demo/Demo/Models/ClientIDRequest.swift (hunk 1 above)
Comment (KunJeongPark) [reply Demo/Demo/Models/ClientIDRequest.swift]: Addressed in commit 8e0b7395b40d47023968064c74cd7ea89e7b78ec

File: Demo/Demo/Models/ClientIDResponse.swift (hunk 2 above)
Comment (KunJeongPark) [reply Demo/Demo/Models/ClientIDResponse.swift]: addressed in commit 8e0b7395b40d47023968064c74cd7ea89e7b78ec

File: Demo/Demo/Networking/DemoMerchantAPI.swift (hunk 3 above)
Comment (KunJeongPark) [reply Demo/Demo/Networking/DemoMerchantAPI.swift]: addressed in commit 8e0b7395b40d47023968064c74cd7ea89e7b78ec

File: Demo/Demo/SwiftUIComponents/SwiftUINativeCheckoutDemo.swift [hunk 4]
```diff
@@ -19,11 +19,13 @@ struct SwiftUINativeCheckoutDemo: View {
     var body: some View {
         switch viewModel.state {
         case .initial:
-            getAccessTokenView()
+            getClientIDView()
         case .loading(let content):
             loadingView(content)
         case let .mainContent(title, content, isFlowComplete):
             checkoutView(title, content, isFlowComplete)
+        case .error(message: let message):
```
Comment (jaxdesmarais) [inline Demo/Demo/SwiftUIComponents/SwiftUINativeCheckoutDemo.swift]: nit: can we do this here to match the other cases?
```suggestion
        case .error(let message):
```

**Review by jaxdesmarais — APPROVED** [review]
👻

File: Demo/Demo/SwiftUIComponents/SwiftUINativeCheckoutDemo.swift (hunk 4 above)
Comment (KunJeongPark) [reply Demo/Demo/SwiftUIComponents/SwiftUINativeCheckoutDemo.swift]: resolved in b5c22856e90afa14d3db9ed92cee1aeeab3a8002

**Review by jcnoriega — APPROVED** [review]

**Comment by scannillo** [comment]
👋 @KunJeongPark - can you make a JIRA for the task of updating our integration guides hosted in this repo (README & docs/)?[ The Android equivalent ](https://github.com/paypal/Android-SDK/pull/146)of this PR included those changes, while they still need to be added into iOS in order for us to release these changes.

**Pull request merged**